flink-table/flink-table-common/src/main/java/org/apache/flink/table/sources/PartitionableTableSource.java

```diff
@@ -43,17 +42,6 @@
 	 */
 	List<Map<String, String>> getPartitions();
```

Comment:
Should we also remove this method? Because all the partition specs should also go through `Catalog`? i.e. `Catalog#listPartitions`